fossunited/fossunited/doctype/foss_event_cfp_submission/templates/foss_event_cfp_submission.html

```diff
@@ -0,0 +1,447 @@
+{% extends "templates/foss_base.html" %}
+
+{% block title %}{{ doc.talk_title }}{% endblock %}
+
+{% block page_content %}
+
+{% set cfp = frappe.db.get_value("FOSS Event CFP", linked_cfp, ["chapter", "anonymous_proposals", "allow_cfp_edit"], as_dict=True) %}
+
+<div class="container" id="head-cfp">
+    <div class="my-6">
+        {{ BreadCrumb(doc.route) }}
+    </div>
+
+    <div class="date-time-category-flex">
+        <div class="icon-text-container">
+            <img src="/assets/fossunited/icons/hash-02.svg">
+            {{ doc.category }}
+        </div>
+        <div class="icon-text-container">
+            <img src="/assets/fossunited/icons/hourglass-02.svg">
+            {{ doc.talk_duration }}
+        </div>
+        {% if doc.first_talk == "Yes" %}
+        <div class="icon-text-container">
+            <img src="/assets/fossunited/icons/stars.svg">
+            {{ _("First Talk") }}
+        </div>
+        {% endif %}
+    </div>
+
+    <h3 class="mt-2 py-2 ">
+        {{ doc.talk_title }}
+    </h3>
+
+    <div class="date-time-category-flex" style="font-size: 1rem;">
+        {% if cfp.anonymous_proposals == 0 %}
+        <div class="gray-600">
+            {{ _("By") }} {{ doc.full_name }} 
+        </div>
+        <div class="gray-600">
+            |
+        </div>
+        {% endif %}
+        <div class="d-flex icon-text-container" id="likes-section">
+            {% set likes = get_doc_likes("FOSS Event CFP Submission", doc.name) %}
+            <div class=" d-flex align-items-center like-button {% if frappe.session.user in likes %} liked {% endif %}" type="button" onClick="toggle_likes('{{doc.name}}' {% if frappe.session.user not in likes %}, 'Yes' {% endif %})">
+                {% include "public/icons/heart.svg" %}
+            </div>            
+            {% if likes %}
+                {{ likes | length }}
+            {% else %}
+                {{ _("0") }}
```

Comment:
```suggestion
                0
```

Why translate a number, static too